EpinelPS/LobbyServer/Msgs/Stage/ClearStage.cs

```diff
@@ -309,5 +311,45 @@ private static void DoQuestSpecificUserOperations(Database.User user, int cleare
             }
             // TODO: add neon
         }
+
+        private static void CreateClearInfo(Database.User user)
+        {
+            NetStageClearInfo clearInfo = new NetStageClearInfo
+            {
+                User = new NetWholeUserData
```

Comment:
You can use LobbyHandler.CreateWholeUserDataFromDbUser instead of creating a new user object